### PR TITLE
Desktop GUI

### PR DESCRIPTION
**Pull request by ocsmit**
Standalone desktop GUI for ProjPicker written in WxPython.

### CODE REVIEW

**Comment by HuidaeCho** [comment]
Still work in progress?

File: guis/desktop/gui.py [hunk 1]
```diff
@@ -0,0 +1,281 @@
+#!/usr/bin/env python3
+################################################################################
+# Project:  ProjPicker (Projection Picker) Desktop GUI
+#           <https://github.com/HuidaeCho/projpicker>
+# Authors:  Owen Smith, Huidae Cho
+#           Institute for Environmental and Spatial Analysis
+#           University of North Georgia
+# Since:    May 27, 2021
```
Comment (HuidaeCho) [inline guis/desktop/gui.py:8]: June 30, 2021

File: guis/desktop/gui.py [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+################################################################################
+import json
+import textwrap
+import pprint
+import wx
+import wx.html2
+from pathlib import Path
+import projpicker as ppik
```
Comment (HuidaeCho) [inline guis/desktop/gui.py:32]: As the last import? As is, it just feels weird.

File: guis/desktop/gui.py [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+
+    def flip(self):
+        """
+        Switch lat lon
+        """
+        corrected_coors = []
+        if self.type == 'Point':
+            corrected_coors = self.coors[1], self.coors[0]
+        else:
+            self.coors = self.coors[0]
```
Comment (HuidaeCho) [inline guis/desktop/gui.py:57]: Trying to understand this line. Passing only the first item to `self.coors`?

Comment (HuidaeCho) [reply guis/desktop/gui.py:57]: Are you using the left `self.coors` as the pointer to the memory space of the whole `self.coors`? Possible in Python?.. thinking..

File: guis/desktop/gui.py [hunk 4]
```diff
@@ -0,0 +1,281 @@
...
+            for i in self.coors:
+                corrected_coors.append(i[::-1])
+        self.coors = list(corrected_coors)
+
+
+#################################
+# GUI
+class ProjpickerGui(wx.Frame):
```
Comment (HuidaeCho) [inline guis/desktop/gui.py:65]: `ProjPickerGUI` or `ProjPickerGui`? What's the Python standard for long acronyms?

File: guis/desktop/gui.py [hunk 5]
```diff
@@ -0,0 +1,281 @@
...
+        btm_left.Add(self.btn_cancel, 1, wx.LEFT | wx.RIGHT, width)
+        self.left.Add(btm_left, 0, wx.BOTTOM)
+
+
+    #################################
+    # RIGHT
+    def crs_info(self, text):
+        # CRS INFO
+        # Set static box
+        crs_info_box = wx.StaticBox(self.panel, 0, "CRS Info")
```
Comment (HuidaeCho) [inline guis/desktop/gui.py:151]: I would make it as a read-only textbox so we can copy its content? Possible?

Comment (HuidaeCho) [reply guis/desktop/gui.py:151]: Not aligned, not copyable.

File: guis/desktop/gui.py [hunk 6]
```diff
@@ -0,0 +1,281 @@
...
+        # Set sizer
+        browser_size = wx.BoxSizer(wx.HORIZONTAL)
+        self.right.Add(self.browser, 1, wx.EXPAND | wx.ALL, 10)
+
+
+    def vertices_alert(self):
+        wx.MessageBox("Too many vertices, please delete geometry.")
+
+
+    def __crs_string(self, crs: list):
```
Comment (HuidaeCho) [inline guis/desktop/gui.py:188]: Why underscores?

**Comment by HuidaeCho** [comment]
@ocsmit Can I merge it and start working on it now? This looks very cool! Awesome job.

**Comment by HuidaeCho** [comment]
Oh! From your email, you're adding more features. Let's merge it now and add features as separate PRs, if you're fine with that.

**Comment by ocsmit** [comment]
> 
> 
> Oh! From your email, you're adding more features. Let's merge it now and add features as separate PRs, if you're fine with that.

That's not a problem!

**Pull request merged**